src/app/components/job-list/job-list.component.ts

```diff
@@ -12,22 +13,33 @@ import { WeatherService } from 'src/app/services/weather.service';
   styleUrls: ['./job-list.component.scss']
 })
 export class JobListComponent implements OnInit, OnDestroy {
-  // TODO - Pick locations that will have alerts prior to meeting!
-  public jobs: JobData[];
+  // public jobs: JobData[];
```

Comment:
remove comment

Reply:
Resolved. Removed unnecessary comments, and moved code from ngOnInit to wrapper method.